ca/certificate-authority.go

```diff
@@ -131,7 +95,7 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
-	issuer, err := loadIssuer(issuerCert)
+	issuer, err := loadIssuer(commonConfig.IssuerCert)
```

Comment:
Just pass in the IssuerCert stuff that it needs, not the whole config.

In fact, this probably should be injected in, but that's not for this review.